From a2f8ba8e4795d65eefdd8e42529d88eb1fdf21ff Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sun, 5 Sep 2010 14:34:40 +0000
Subject: [PATCH 13/15] gettextize: git-revert messages using the "me" variable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Mark messages that used the "me" variable for translation. The
potential values of the variable had already been marked using N_(),
so we can use _(me) here.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/revert.c              |   14 +++++++-------
 t/t3501-revert-cherry-pick.sh |    2 +-
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/builtin/revert.c b/builtin/revert.c
index 2f319f8..8dff244 100644
--- a/builtin/revert.c
+++ b/builtin/revert.c
@@ -286,10 +286,10 @@ static NORETURN void die_dirty_index(const char *me)
 		die_resolve_conflict(me);
 	} else {
 		if (advice_commit_before_merge)
-			die("Your local changes would be overwritten by %s.\n"
-			    "Please, commit your changes or stash them to proceed.", me);
+			die(_("Your local changes would be overwritten by %s.\n"
+			    "Please, commit your changes or stash them to proceed."), _(me));
 		else
-			die("Your local changes would be overwritten by %s.\n", me);
+			die(_("Your local changes would be overwritten by %s.\n"), _(me));
 	}
 }
 
@@ -339,7 +339,7 @@ static int do_recursive_merge(struct commit *base, struct commit *next,
 	if (active_cache_changed &&
 	    (write_cache(index_fd, active_cache, active_nr) ||
 	     commit_locked_index(&index_lock)))
-		die("%s: Unable to write new index file", me);
+		die(_("%s: Unable to write new index file"), _(me));
 	rollback_lock_file(&index_lock);
 
 	if (!clean) {
@@ -446,8 +446,8 @@ static int do_pick_commit(void)
 		return fast_forward_to(commit->object.sha1, head);
 
 	if (parent && parse_commit(parent) < 0)
-		die("%s: cannot parse parent commit %s",
-		    me, sha1_to_hex(parent->object.sha1));
+		die(_("%s: cannot parse parent commit %s"),
+		    _(me), sha1_to_hex(parent->object.sha1));
 
 	if (get_message(commit->buffer, &msg) != 0)
 		die(_("Cannot get commit message for %s"),
@@ -570,7 +570,7 @@ static int revert_or_cherry_pick(int argc, const char **argv)
 	}
 
 	if (read_cache() < 0)
-		die("git %s: failed to read the index", me);
+		die(_("git %s: failed to read the index"), _(me));
 
 	prepare_revs(&revs);
 
diff --git a/t/t3501-revert-cherry-pick.sh b/t/t3501-revert-cherry-pick.sh
index bc7aedd..b12b1b4 100755
--- a/t/t3501-revert-cherry-pick.sh
+++ b/t/t3501-revert-cherry-pick.sh
@@ -81,7 +81,7 @@ test_expect_success 'revert after renaming branch' '
 
 '
 
-test_expect_success 'revert forbidden on dirty working tree' '
+test_expect_success NO_GETTEXT_POISON 'revert forbidden on dirty working tree' '
 
 	echo content >extra_file &&
 	git add extra_file &&
-- 
1.7.2.3.313.gcd15

